Sources/NIOHTTPClient/HTTPHandler.swift

```diff
@@ -19,24 +19,40 @@ import NIOHTTP1
 import NIOSSL
 
 public extension HTTPClient {
-    enum Body: Equatable {
-        case byteBuffer(ByteBuffer)
-        case data(Data)
-        case string(String)
-
-        var length: Int {
-            switch self {
-            case .byteBuffer(let buffer):
-                return buffer.readableBytes
-            case .data(let data):
-                return data.count
-            case .string(let string):
-                return string.utf8.count
+    typealias ChunkProvider = (@escaping (IOData) -> EventLoopFuture<Void>) -> EventLoopFuture<Void>
```

Comment:
Can/should we lift this `typealias` inside the `struct Body`?

Reply:
good idea, fixed, thanks!